### PR TITLE
python3Packages.elementpath: 2.2.1 -> 2.2.2

### PR DESCRIPTION
**Pull request by jonringer**
###### Motivation for this change
routine bump

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jonringer** [comment]
thought there would be more rebuilds

```
https://github.com/NixOS/nixpkgs/pull/123862

13 packages built:
banking cyclonedx-python matrix-synapse python38Packages.elementpath python38Packages.fints python38Packages.pysaml2 python38Packages.sepaxml python38Packages.xmlschema python39Packages.elementpath python39Packages.fints python39Packages.pysaml2 python39Packages.sepaxml python39Packages.xmlschema
```

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123862` at b24e9df2 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>13 packages built successfully:</summary>
  <ul>
    <li>banking</li>
    <li>cyclonedx-python</li>
    <li>matrix-synapse</li>
    <li>python38Packages.elementpath</li>
    <li>python38Packages.fints</li>
    <li>python38Packages.pysaml2</li>
    <li>python38Packages.sepaxml</li>
    <li>python38Packages.xmlschema</li>
    <li>python39Packages.elementpath</li>
    <li>python39Packages.fints</li>
    <li>python39Packages.pysaml2</li>
    <li>python39Packages.sepaxml</li>
    <li>python39Packages.xmlschema</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/stale-substitute.md">stale-substitute</a>

Stale substituteInPlace detected.
substituteStream(): WARNING: pattern 'elementpath~=2.0.0' doesn't match anything in file 'setup.py'
Near pkgs/development/python-modules/xmlschema/default.nix:39:0:
```
   |
39 |     description = "XML Schema validator and data conversion library for Python";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/xmlschema/default.nix [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
 
   src = fetchFromGitHub {
     owner = "sissaschool";
     repo = "xmlschema";
     rev = "v${version}";
-    sha256 = "0pv8zdz03wjnjwrgjr5pc3q69h7zh51h0iwgwxwl65qi1r6ydk54";
+    sha256 = "sha256-GL2PlHxKDSEsZwHPBAy+tjBSbKyvlbXUWwXakKPmzSs=";
   };
 
   propagatedBuildInputs = [ elementpath ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/xmlschema/default.nix:18]: Please remove the postPatch.

**Comment by jonringer** [comment]
forgot that I rebased on the PR. sorry for spam

**Comment by jonringer** [comment]
```
https://github.com/NixOS/nixpkgs/pull/123862

14 packages built:
banking cyclonedx-python matrix-synapse matrix-synapse-plugins.matrix-synapse-mjolnir-antispam python38Packages.elementpath python38Packages.fints python38Packages.pysaml2 python38Packages.sepaxml python38Packages.xmlschema python39Packages.elementpath python39Packages.fints python39Packages.pysaml2 python39Packages.sepaxml python39Packages.xmlschema
```

**Pull request merged**